lldb/tools/lldb-dap/DAP.h

```diff
@@ -406,6 +419,10 @@ struct DAP {
   std::thread progress_event_thread;
   /// @}
 
+  /// list of addresses mapped by sourceReference(index - 1)
```

Comment:
`index - 1` seems wrong? That would mean that the address at index 0 has a source reference of -1?

```suggestion
  /// List of addresses mapped by sourceReference.
```